cmd/controller/app/server.go

```diff
@@ -58,6 +58,12 @@ var (
 	scheme = runtime.NewScheme()
 )
 
+const (
+	// deployModeAnnotationKey describe the klusterlet deploy mode when importing a managed cluster.
+	deployModeAnnotationKey string = "import.open-cluster-management.io/klusterlet-deploy-mode"
```

Comment:
It's seems every addon need to define those vars.
Is there a way to make a consist pkg in addon-framework and addons can import and use?
[Not a blocker, it's ok in either way]

Reply:
this annotation is only used in ACM, so I think there is not a good reason to put it in addon-framework.